src/Bridges/Constraint/bridges/functionize.jl

```diff
@@ -33,6 +33,15 @@ function MOI.get(
     return MOI.get(model, attr, bridge.constraint)
 end
 
+# Needed to avoid ambiguity
+function MOI.get(
+    model::MOI.ModelLike,
+    ::MOI.CanonicalConstraintFunction,
+    bridge::AbstractFunctionConversionBridge,
+)
+    return MOI.get(model, MOI.ConstraintFunction(), bridge)
```

Comment:
How do we know the constraint is canonical ?

Reply:
Because there are only two bridges, and they both go back to `VariableIndex` and `VectorOfVariables`. I guess I could add back the `canonical()` for clarity.